### ISSUE TITLE
Breaking a shipwreck chest that contains a treasure map

### ISSUE THREAD

**Issue by rvbsm** [thebrightspark/AsyncLocator#8]
Breaking a shipwreck chest that contains a not processed treasure map will drop a working map, and once the player collects it, it cannot be processed or dropped/moved. Server restart doesn't fix it.

![image](https://github.com/thebrightspark/AsyncLocator/assets/39232658/0c1d99f9-20f9-413e-81fb-1a8ac5f9c92d)

**Comment by thebrightspark** [comment]
Yeah, I've known this would be an issue but ignored it at the time in the hopes people prefer to look in the chests 😅 I knew it'd be a bit of work to get it supported.

I'll see if I can support breaking chests too

**Comment by thebrightspark** [comment]
In the mean time, if you are able to run commands you can spawn yourself a new chest and open it to properly generate a map:
`/setblock ~1 ~ ~ chest{LootTable:"chests/shipwreck_map"}`

**Comment by rvbsm** [comment]
> I've known this would be an issue but ignored it at the time in the hopes people prefer to look in the chests 😅

Very vital 😅

> In the mean time, if you are able to run commands you can spawn yourself a new chest and open it to properly generate a map

Yeah, thank you! Of course, I'd rather find another shipwreck or not do this at all, but thanks for hinting
